### PR TITLE
Features/checkbox

### PR DESCRIPTION
**Pull request by yinchyy**

### CODE REVIEW

File: src/Checkbox/checkbox.module.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+input[type="checkbox"] {
+  appearance: none;
+  width: 24px;
+  height: 24px;
+  outline: 1px solid $color-dark;
+  border-radius: 6px;
+  cursor: pointer;
+  display: inline-block;
+  &:checked{
+    background-image: url('iconoir/icons/check.svg');
```
Comment (pilotpirxie) [inline src/Checkbox/checkbox.module.scss:13]: does it work when bundled (yarn build && yarn link)?

File: src/config/backgrounds.scss [hunk 2]
```diff
@@ -28,6 +28,7 @@
     background-color: rgba(0, 0, 0, 0);
     color: $bgColor;
     text-decoration: underline;
+    padding: 5px;
```
Comment (pilotpirxie) [inline src/config/backgrounds.scss:31]: why 5px?

**Review by pilotpirxie — APPROVED** [review]

File: src/Checkbox/checkbox.module.scss (hunk 1 above)
Comment (yinchyy) [reply src/Checkbox/checkbox.module.scss]: It didn't, but after [a64b9fc](https://github.com/pilotpirxie/react-toucan-ui/pull/5/commits/a64b9fc40554ea76c2111cd2cd9d13c0cbae464b) it does

File: src/config/backgrounds.scss (hunk 2 above)
Comment (yinchyy) [reply src/config/backgrounds.scss:31]: because UX demanded less gap when link style is selected, but didn't provide any value

**Pull request merged**